doc/news/changes/minor/20230611HeinzMunch

```diff
@@ -0,0 +1,4 @@
+Fixed: Renumbering in FEPointEvaluation was computed wrong for multiple components
+and when used with first_selected_component!=0.
```

Comment:
Can we improve the language a bit? Also, I think the user will not necessarily be aware what this means exactly, so I suggest:
```suggestion
Fixed: The numbering applied within FEPointEvaluation did not work correctly
for multiple components and when used with first_selected_component!=0 for
certain FiniteElement types, leading to wrong results. This is now fixed.
```

Reply:
For the changelog between releases I think we only need to mention that `integrate_fast()` was not working correctly in the last release for `first_selected_component!=0`. The other bug in `evaluate_fast()` was only recently introduced by https://github.com/dealii/dealii/pull/15274 .